packages/core/content-manager/server/controllers/collection-types.js

```diff
@@ -329,6 +329,50 @@ module.exports = {
 
     const number = await entityManager.getNumberOfDraftRelations(id, model);
 
+    return {
+      data: number,
+    };
+  },
+  async getMultipleEntriesNumberOfDraftRelations(ctx) {
+    const { userAbility } = ctx.state;
+    const { model, ids } = ctx.params;
+
+    const entityManager = getService('entity-manager');
+    const permissionChecker = getService('permission-checker').create({ userAbility, model });
+
+    if (permissionChecker.cannot.read()) {
+      return ctx.forbidden();
+    }
+
+    const permissionQuery = await permissionChecker.sanitizedQuery.read(ctx.query);
+    const populate = await getService('populate-builder')(model)
+      .populateFromQuery(permissionQuery)
+      .build();
+
+    const idsArray = JSON.parse(ids);
+
+    if (idsArray.length <= 0) {
+      return {
+        data: 0,
+      };
+    }
+
+    let number = 0;
+
+    for (const id of idsArray) {
+      const entity = await entityManager.findOne(id, model, { populate });
+
+      if (!entity) {
+        return ctx.notFound();
+      }
+
+      if (permissionChecker.cannot.read(entity)) {
+        return ctx.forbidden();
+      }
+
+      number += await entityManager.getNumberOfDraftRelations(id, model);
```

Comment:
I think there could be a new method on the entity manager to handle multiple ids. In that method you could findMany where ids, as Marc suggested. I think then, a more functional approach would be to use Array.reduce with sumDraftCounts to get the number.

Reply:
I would say that that should be a chose done by the user and should not be hardcoded to A or B or we need to make 2 functions

Reply:
@Feranchz do you mean having two different relation attributes that point to the same entity? I would say the best approach would be to say "this entity is related to X number of Drafts" and not count duplicates.

But the implementation of doing that could be a bit more extense of what we have now.

Reply:
Do we want to show the concrete Draft entities when bulk publishing in the future?